sklearnex/_device_offload.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, filter_string):
             self._filter_string = filter_string
             self.is_cpu = 'cpu' in filter_string
             self.is_gpu = 'gpu' in filter_string
+            self.has_aspect_fp64 = self.is_cpu
```

Comment:
I don't like these hardcoded restriction.
At least comment is required.

Reply:
Comment was added